dpe/src/dpe_instance.rs

```diff
@@ -33,8 +33,8 @@ pub struct DpeInstance<'a> {
     /// `InitializeContext(simulation=false)`) once per reset cycle.
     pub(crate) has_initialized: bool,
 
-    // Issuer Common Name to use in DPE leaf certs
-    pub(crate) issuer_cn: &'a [u8],
+    // DER encoded Issuer Common Name to use in DPE leaf certs
+    pub(crate) issuer_der: &'a [u8],
```

Comment:
I wonder if we should make this a platform API instead of passing to the DpeInstance constructor. The reason is that the platform may be dynamically encoding it (that's what Caliptra will probably do, because the issuer name can change). So if we pass it in, the lifetime stuff could get tricky.

For a platform API, I'm thinking something like

```rust

struct DerName {
  der: [u8; Self::MAX_ISSUER_SIZE];
  len: usize;
}

impl DerName {
  const MAX_ISSUER_SIZE = 256; // Maybe?
}

fn get_encoded_issuer() -> Result<DerName, PlatformError>;
```